tools/WixBuild.props

```diff
@@ -184,7 +192,22 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VCPath>$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC', 'ProductDir', null, RegistryView.Registry64, RegistryView.Registry32))</VCPath>
+    <VS2017Path Condition=" '$(VS2017Path)'=='' and '$(VsInstallRoot)'!=''">$(VsInstallRoot)\</VS2017Path>
```

Comment:
True. (Well, presumably; I mean, Windows 10 is the "last" Windows, right? 😈 ) There are MSBuild properties we can rely on, assuming MSBuild versions as you'd expect.